src/openrct2/ride/Ride.cpp

```diff
@@ -2386,8 +2386,10 @@ static void ride_inspection_update(Ride* ride)
         ride->last_inspection--;
 
     int32_t inspectionIntervalMinutes = RideInspectionInterval[ride->inspection_interval];
-    if (inspectionIntervalMinutes == 0)
+    if (inspectionIntervalMinutes == 0) {
```

Comment:
Curly brace should be on the next line. CI will even fail because of it.

Also, I think it would be better if it also cancelled any mechanic currently heading for inspection.

Reply:
Ah, good catch. Changed the brace.
The inspection status update only gets run every 51 seconds, so it may take a while for the flag to get cleared. As soon as the flag is cleared the mechanic will stop heading for the ride to be inspected.